README.md

```diff
@@ -602,6 +601,15 @@ VPC CNI uses `iptables-legacy` by default. Setting `ENABLE_NFTABLES` to `true` w
 **Note:** VPC CNI image contains `iptables-legacy` and `iptables-nft`. Switching between them is done via `update-alternatives`. It is *strongly* recommended that the iptables mode matches that which is used by the base OS and `kube-proxy`.
 Switching modes while pods are running or rules are installed will not trigger reconciliation. It is recommended that rules are manually updated or nodes are drained and cordoned before updating. If reloading node, ensure that previous rules are not set to be persisted.
 
+#### `AWS_EXTERNAL_SERVICE_CIDRS` (v1.13.0+)
+
+Type: String
+
+Default: empty
+
+Specify a comma-separated list of IPv4 and/or IPv6 CIDRs that *must* be routed via main routing table. This is required for secondary ENIs to reach endpoints outside of VPC that are backed by a service.
```

Comment:
We don't need to support this for IPv6. We currently only support IPv6 in PD mode and so we will only need one IPv6 prefix attached to Primary ENI for Pod IPs and so we will not be affected by symmetric routing fix in IPv6 mode.